### PR TITLE
Rework Logs content and Format

### LINKED ISSUE #207: Rework logs
### Explain problem to solve

We have to rework the logs because there are too many and we have more describing logs instead of meaning.

### Describe the solution you would like

We can:
- Remove # on logs

For example :
-  (On Volume) specify how many volume instead of "find volume"
-  (On vm) Refactor because we have  "Vm stopped state VmStateStopping"
- (On subnet) "Get lb subnetId", maybe it is better to set it after the subnetId, err := getSubnetResourceId(subnetName, clusterScope) (same for the other)
- ...

(https://github.com/outscale-dev/cluster-api-provider-outscale/pull/179)

### Additional context

None

### Environment

```markdown
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):
- Kernel (e.g. `uname -a`):
- cluster-api-provider-outscale version:
- cluster-api version:
- Install tools:
- Kubernetes Distribution:
- Kubernetes Diestribution version:
```

### PR DESCRIPTION
**Pull request by outscale-vbr**
**What type of PR is this?**
/kind cleanup  

**What this PR does / why we need it**:
Rework logs and improve logs format
Close #207 
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #207 

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

### CODE REVIEW

**Comment by outscale-vbr** [comment]
https://github.com/outscale/cluster-api-provider-outscale/pull/221 will be merge first to handle cve.
and change on trivyignore will be removed for this PR.

File: helm/clusterapioutscale/values.yaml [hunk 1]
```diff
@@ -8,7 +9,15 @@ clusterapioutscale:
...
-
+    # integer in second defining initial duration
+    # -- Initial duraction of backoff
+    backoff_duration: "1"
+    # float define the factor multiplied by Duration each iteration
+    # -- Factor multiplied by Duration for each iteration
+    backoff_factor: "1.0"
+    # integer defining the remaining number of iterations in which the duration parameter may change
+    # -- Remaining number of iterations in which the duration parameter may change
+    backoff_steps: "1"
```
Comment (outscale-mdr) [inline helm/clusterapioutscale/values.yaml:20]: Another PR ? or commit ?

File: capm.yaml [hunk 2]
```diff
@@ -1443,6 +1451,12 @@ spec:
               key: region
               name: cluster-api-provider-outscale
               optional: true
+        - name: BACKOFF_DURATION
+          value: "1"
+        - name: BACKOFF_FACTOR
+          value: "1.0"
+        - name: BACKOFF_STEPS
+          value: "1"
```
Comment (outscale-mdr) [inline capm.yaml:1459]: Another PR ? Which issue ?

**Review by outscale-mdr — CHANGES_REQUESTED** [review]

File: capm.yaml (hunk 2 above)
Comment (outscale-vbr) [reply capm.yaml:1459]: Another commit

File: helm/clusterapioutscale/values.yaml (hunk 1 above)
Comment (outscale-vbr) [reply helm/clusterapioutscale/values.yaml:20]: another commit

**Comment by outscale-vbr** [comment]
@outscale-mdr Are you ok with this PR ?

File: capm.yaml [hunk 3]
```diff
@@ -1422,6 +1429,7 @@ spec:
         - --health-probe-bind-address=:8081
         - --metrics-bind-address=127.0.0.1:8080
         - --leader-elect
+        - --zap-log-level=10
```
Comment (outscale-mdr) [inline capm.yaml:1432]: Still positive integer ?

File: config/default/manager_auth_proxy_patch.yaml [hunk 4]
```diff
@@ -32,3 +32,4 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--zap-log-level=10"
```
Comment (outscale-mdr) [inline config/default/manager_auth_proxy_patch.yaml:35]: Same

File: capm.yaml (hunk 3 above)
Comment (outscale-vbr) [reply capm.yaml:1432]: Yes

File: config/default/manager_auth_proxy_patch.yaml (hunk 4 above)
Comment (outscale-vbr) [reply config/default/manager_auth_proxy_patch.yaml:35]: Yes

**Review by outscale-mdr — APPROVED** [review]

**Pull request merged**